.github/workflows/stale.yaml

```diff
@@ -0,0 +1,31 @@
+---
+
+name: Manage stale issues
+
+on:
+  schedule:
+    # run hourly
+    - cron: "0 * * * *"
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: none
+
+    steps:
+      - name: Close Stale Issues
+        uses: actions/stale@v4.1.0
+        with:
+          days-before-stale: 7
+          days-before-close: 7
+          exempt-issue-labels: 'internal,BUG,help wanted,#triage/accepted,#needs-triage'
```

Comment:
Please add ‘Feature’

Reply:
Not sure if case sensitive

Reply:
Apparently, Github labels are case insensitive:

![image](https://user-images.githubusercontent.com/16166434/147834790-bcb47779-8a5b-4e35-9446-52fd01991c1e.png)